mava/utils/logger.py

```diff
@@ -281,6 +302,18 @@ def _make_multi_logger(cfg: DictConfig) -> BaseLogger:
     loggers: List[BaseLogger] = []
     unique_token = datetime.now().strftime("%Y%m%d%H%M%S")
 
+    if (
+        cfg.logger.use_neptune
+        and cfg.logger.use_json
+        and cfg.logger.kwargs.upload_json_data
+        and cfg.logger.kwargs.json_path
+    ):
+        raise ValueError(
```

Comment:
Maybe instead of raising an error, we raise a warning and change the use_json to empty str?

Reply:
I'm not super sure about this because it could be that someone is doing a big run on a some local hardware where they just want the single json file at the end. In which case it makes more sense to not upload json files instead of storing many.

Reply:
I see! yeah this is reasonable

Reply:
I also made the error message a bit more descriptive :)